src/Compilers/CSharp/Portable/Generated/CSharp.Generated.g4

```diff
@@ -880,8 +880,8 @@ this_expression
 interpolated_string_expression
   : '$"' interpolated_string_content* '"'
   | '$@"' interpolated_string_content* '"'
-  | interpolated_multi_line_raw_string_start_token interpolated_string_content* interpolated_multi_line_raw_string_end_token
-  | interpolated_single_line_raw_string_start_token interpolated_string_content* interpolated_single_line_raw_string_end_token
+  | interpolated_multi_line_raw_string_start_token interpolated_string_content* interpolated_raw_string_end_token
```

Comment:
I would be good to update the grammar in spec to match.
[https://github.com/dotnet/csharplang/blob/main/proposals/raw-string-literal.md](https://github.com/dotnet/csharplang/blob/main/proposals/raw-string-literal.md)

Reply:
will do.